types/styletron-react/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "@types/styletron-standard": "^1.0.0"
```

Comment:
Is this the right way of doing this? This package is currently in #35301. I'm not sure what the initial version will be, but I assume 1.0.0?